src/encoded/static/components/datatable.js

```diff
@@ -139,7 +139,7 @@ const DataTable = ({ tableData }) => {
 
                                 // Cell's colSpan can be a specific number, 0 (full width), or
                                 // undefined (default of 1).
-                                let cellColSpan = (typeof cell === 'object') && cell.colSpan;
+                                let cellColSpan = (typeof cell === 'object' && cell !== null) && cell.colSpan;
```

Comment:
(Minor) Out of curiosity, why are you comparing cell to object; why not something like Node or Element?

Reply:
The fix is that in Javascript, `typeof null` returns “object.” I added the `cell !== null` to take care of that case.